### ISSUE TITLE
[Bug]: Installing sd-transforms breaks style-dictionary functionality

### ISSUE THREAD

**Issue by olliequ** [tokens-studio/sd-transforms#197]
### What happened?

If you install sd-transforms (using `npm install @tokens-studio/sd-transforms`) style-dictionary will instantly stop working with the following error:

`TypeError: styleDictionary.buildAllPlatforms is not a function`

This is without even changing the SD config file. All I've done is install sd-transforms.

My SD config file (config.js) looks like this:

```
module.exports = {
  // This array of globs will match our token source files
  source: ["input/**/*.json"],
  platforms: {
    scss: {
      transformGroup: 'scss',
      buildPath: "output/scss/",
      files: [
        {
          destination: "_variables.scss",
          format: "scss/variables",
        },
      ],
    },
    css: {
      transformGroup: 'css',
      buildPath: "output/css/",
      files: [
        {
          destination: "_variables.css",
          format: "css/variables",
        },
      ],
    }
  },
}```

### Reproduction

I'm on:

@tokens-studio/sd-transforms@0.11.1
style-dictionary@3.8.0

### Expected output

My css files to build, hopefully being able to use sd-transforms!

### Version

0.11.1

**Comment by jorenbroekema** [comment]
Huh that's odd, I reproduced it, looking into it, thanks for reporting!

**Comment by jorenbroekema** [comment]
Turns out that in my browser-style-dictionary patch I created a regression where CommonJS (`module.exports = {...}`) file configs for style-dictionary doesn't really work anymore I'm afraid. I'll see if I can find a way to fix it somehow

**Comment by jorenbroekema** [comment]
fixed in v0.11.8. Might need to clean out your node_modules folder or package-lock file to ensure browser-style-dictionary is no longer installed as a dependency of sd-transforms.
